config/35-pilot.ini

```diff
@@ -3,15 +3,15 @@
 ;===================================================================
 
 ; For each pilot type, add a new pilot section.
-; Each pilot name must be unique for the entire grid, so make sure to not
-; pick anything generic like "MAIN". The name will be used as-is as the
+; The name of each pilot type must be unique for the entire grid, so make sure
```

Comment:
Why must it be unique? And it would be great if we could provide a little more guidance on this

Reply:
It's used in the `Name` attribute of the `OSG_ResourceCatalog` attribute that gets sent by the CE to the central collector -- just like `Subcluster` and `Resource Entry` sections. It's not absolutely necessary for it to be unique -- ads from different CEs won't stomp on each other -- but uniqueness makes it easier to query.

The advice I recommended in the doc is that the name should start with the name of the CE's topology resource; I can update the comment to say the same.

We could also make osg-configure prepend the resource name to the pilot type name (so if you have `resource=ITB-CE1` in the `Site Information` section, and name this section `Pilot 4CPU`, then the Name attribute in the OSG_ResourceCatalog entry would be `ITB-CE1-4CPU`. I can't remember if we discussed that or not...

Reply:
I vote that we remove the global requirement for `Pilot` and whatever the non-ATLAS one is between `Subcluster` and `Resource Entry` and just insist that they're unique among entries for that CE. For our own queries, it's easy enough to find unique entries based on the combination of `OSG_Resource` and pilot entry name in the schedd ad. It's much harder for users to make sure their name is unique across the entire OSG.

I think concatenating the resource and pilot entry names from the config to create the advertised pilot name is nice but unnecessary given the above.

Reply:
OK. (I think the global uniqueness constraint was for BDII back in the day and we just kept it because we keep things.)

Reply:
Updated.